tests/cypress/integration/actions_projects/case_94_move_task_between_projects.js

```diff
@@ -0,0 +1,107 @@
+    const imagesFolder = `cypress/fixtures/${imageFileName}`;
+    const directoryToArchive = imagesFolder;
+    const advancedConfigurationParams = false;
+    const forProject = true;
+    const attachToProject = false;
+    const multiAttrParams = false;
+
+    function checkTask (project, expectedResult) {
+        cy.goToProjectsList();
+        cy.openProject(project);
+        cy.get('.cvat-tasks-list-item').should(expectedResult);
+    }
+
+    before(() => {
+        cy.imageGenerator(imagesFolder, imageFileName, width, height, color, posX, posY, firtsProject.label, imagesCount);
+        cy.createZipArchive(directoryToArchive, archivePath);
+        cy.visit('/');
+        cy.login();
+    });
+
+    beforeEach(() => {
+        cy.goToProjectsList();
+        cy.createProjects(firtsProject.name, firtsProject.label, firtsProject.attrName, firtsProject.attrVaue, firtsProject.multiAttrParams);
+        cy.createProjects(secondProject.name, secondProject.label, secondProject.attrName, secondProject.attrVaue, secondProject.multiAttrParams);
+        cy.openProject(firtsProject.name);
+        cy.createAnnotationTask(
+            taskName,
+            firtsProject.label,
+            firtsProject.attrName,
+            firtsProject.attrVaue,
+            archiveName,
+            multiAttrParams,
+            advancedConfigurationParams,
+            forProject,
+            attachToProject.no,
+            firtsProject.name,
+        );
+    });
+
+    afterEach(() => {
+        cy.goToProjectsList();
+        cy.openProject(firtsProject.name);
+        cy.deleteProjectViaActions(firtsProject.name);
+        cy.openProject(secondProject.name);
+        cy.deleteProjectViaActions(secondProject.name);
+    });
+
+    describe(`Testing "Case ${caseID}"`, () => {
+        // Waiting to fix https://github.com/openvinotoolkit/cvat/issues/3281
+        it.skip('Move a task between projects from a project.', () => {
+            checkTask(secondProject.name, 'not.exist');
+            checkTask(firtsProject.name, 'exist');
+            cy.movingTask(taskName, secondProject.name, firtsProject.label, secondProject.label);
+            checkTask(firtsProject.name, 'not.exist');
+            checkTask(secondProject.name, 'exist');
+        });
+
+        it('Move a task between projects from task list.', () => {
+            cy.goToTaskList();
+            cy.movingTask(taskName, secondProject.name, firtsProject.label, secondProject.label);
+            checkTask(firtsProject.name, 'not.exist');
+            checkTask(secondProject.name, 'exist');
+        });
+
+        it('Move a task between projects from a task.', () => {
+            cy.goToTaskList();
+            cy.openTask(taskName);
+            cy.movingTask(taskName, secondProject.name, firtsProject.label, secondProject.label, true);
+            checkTask(firtsProject.name, 'not.exist');
+            checkTask(secondProject.name, 'exist');
+        });
```

Comment:
What is difference with ``case_95...``?

Reply:
In case 94, a task that is already in the project is moved to another project. In case 95, a task that was created outside of the project is moved to the project.